cypress/plugins/index.js

```diff
@@ -0,0 +1,41 @@
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+/**
+ * @param `on` is used to hook into various events Cypress emits
+ * @param `config` is the resolved Cypress config
+ * @return {[type]} [description]
+ */
+
+// Reference: https://docs.cypress.io/api/plugins/configuration-api.html#Promises
+
+/* eslint-disable global-require */
+
+// plugins file
+module.exports = (on, cypressConfig) => {
+  // Load the config
+  if (!process.env.CI) {
+    const path = require('path');
+    const envPath = path.join(__dirname, '../.env');
+    console.log('Loading environment variables from', envPath);
+    require('dotenv').config({ path: envPath });
+  }
+
+  return Object.assign({}, cypressConfig, {
+    env: Object.assign({}, cypressConfig.env, {
+      LOGIN_USER: process.env.CYPRESS_LOGIN_USER,
+      LOGIN_PASSWORD: process.env.CYPRESS_LOGIN_PASSWORD,
+      PROJECT_KEY: process.env.CYPRESS_PROJECT_KEY,
+    }),
+  });
```

Comment:
Simple and good. Wondering if one day we can actually move shared commands etc from the mc-fe here also allowing custom app authors to easily write e2e tests. Even though it ia likely an „unimportant“ feature wish right now. Anyway, it would also force us to have the same/in-sync logic across repos. So that e.g these env vars are read the same on travis and circle no matter what repo. 

These things btw is what I hoped would be nice tasks for the full time focus FLD role.

Reply:
> allowing custom app authors to easily write e2e tests

Yeah, I was also thinking of having a cypress setup in e.g. the example apps. Or have a plugin package that injects commands (`@commercetools-frontend/cypress-mc-app`).

Reply:
Yeah. My head was thinking more of @commercetools-frontend/cypress-mc-app. Maybe create an issue for it so we do not forget and can give it to an FLD who has time to pick it up? As a side effect of that we „force“ to share and gain knowledge of these things.

Reply:
Done: #429